src/components/helpers.tsx

```diff
@@ -0,0 +1,16 @@
+import { css } from "styled-components"
+
+/**
+ * Helper function to display an element as a block that inherits its parents width
+ * @param margin value in pixels to remove from width 100%
+ */
+export const block = (margin: number = 0) => {
+    return props => {
+        if (props.block)
+            return css`
+                display: block;
+                width: calc(100% - ${margin}px);
+                margin: 10px auto;
+            `
+        }
```

Comment:
@l2succes, @alloy - Question - can we settle and enforce spacing? This is the oldest argument out there, but I think it's important for consistency.  

From my own opinion, since so much of our existing codebase -- across Ruby, JS, CoffeeScript, Scala, Python (hokusai) and more -- are 2 spaces, can we keep it that way or at least open up for discussion?